### PR TITLE
Fix integration tests passing `--debug version`

### PR DESCRIPTION
**Pull request by thpani**
Fix 2 integration tests. This addresses two issues with these tests:

* The tests pipe through `sed` to filter `-DTLA-library`, which is not printed.
* Passing `--debug version` does not recognize the version command and prints a usage message.

This PR removes the sed filter and flips the `--debug` and `version` arguments.

### CODE REVIEW

**Review by shonfeder — COMMENTED** [review]
One question...

File: test/tla/cli-integration-tests.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ $ JVM_ARGS="-Xms1m -XX:+UseSerialGC" apalache-mc --debug version | sed 's/-DTLA-
 If we set the max heap size (with `-Xmx`) it will override the default max heap size:
 
 ```sh
-$ JVM_ARGS="-Xmx16m" apalache-mc --debug version | sed 's/-DTLA-Library.*//'
+$ JVM_ARGS="-Xmx16m" apalache-mc version --debug
```
[the remark this answers is not in the document]
Comment (thpani) [reply test/tla/cli-integration-tests.md:109]: You're right, with other commands (tested `--debug` and `--out-dir`) it respects global options irrespective of their placement. So this seems to be a bug with the special handling of the `version` command in [backuity/clist](https://github.com/backuity/clist)?

[the remark this answers is not in the document]
Comment (shonfeder) [reply test/tla/cli-integration-tests.md:109]: Oh! I see. So this is like a design bug with our external dependency.

It's occurred to me that we might enjoy a different CLI parsing lib, but I think that's most yaking shaving at this point :)

So, iiuc, this is about the way `version` is parsed by `clist`, not a problem with our use of the library., Thanks for explaining!

**Review by shonfeder — APPROVED** [review]
LGTM.

Thanks!

File: test/tla/cli-integration-tests.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (thpani) [reply test/tla/cli-integration-tests.md:109]: We should revisit this after #1624.

**Pull request merged**